### PR TITLE
chore: Document `pino-pretty` usage

### PR DESCRIPTION
**Pull request by 72636c**
This powers Pino's `prettyPrint` option and can be useful for local execution.

~~It does bloat the bundle a little.~~ Demoted to a peer dependency to reduce bloat

### CODE REVIEW

**Comment by 72636c** [comment]
I'm thinking that this can be the first proper public release `v4.4.2` from this org, once we sort out publishing

**Comment by kosanna** [comment]
Should it be more of a dev dependency? Seems a bit of an overkill for prod 🤔

**Review by shawn-dsz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -90,7 +89,27 @@ const logger = createLogger(
...
+For example, you can install **[pino-pretty]** as a `devDependency`:
+
+```shell
+yarn add --dev pino-pretty
+```
+
+Then selectively enable pretty printing when running your application locally:
+
+```typescript
+import createLogger from '@seek/logger-js';
```
Comment (runk) [inline README.md:106]: Needs to be updated

**Comment by 72636c** [comment]
This probably isn't a big enough deal to release if we aren't bundling it as a `dependency`

**Pull request merged**